src/configuration/prima.rs

```diff
@@ -77,13 +82,45 @@ impl FromStr for Environment {
     }
 }
 
-impl ToString for Environment {
-    fn to_string(&self) -> String {
+impl Display for Environment {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            Environment::Dev => "dev".to_string(),
-            Environment::Qa => "qa".to_string(),
-            Environment::Staging => "staging".to_string(),
-            Environment::Production => "production".to_string(),
+            Environment::Dev => write!(f, "dev"),
+            Environment::Qa => write!(f, "qa"),
+            Environment::Staging => write!(f, "staging"),
+            Environment::Production => write!(f, "production"),
+        }
+    }
+}
+
+/// Represents the country in which the datadog client runs.
+/// This is useful for enforcing rules based on country for every application that uses the library.
+#[derive(PartialEq, Debug, Clone)]
+pub enum Country {
+    It,
+    Es,
+    Uk,
+}
```

Comment:
Late to the party, but I guess that `Common` could be a possible value here since there could be services deployed once for all countries (Lira will be one of them IIRC).
Could you confirm @pdincau @mbusi @matteosister?

Reply:
Yes! I would call it `global`

Reply:
You are right! Here's the [card](https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PLATFORM-686/PrimaDatadogrs-Add-global-to-list-of-countries)